services/identity/service.go

```diff
@@ -39,14 +43,14 @@ func init() {
 // Service handles identities
 type Service struct {
 	*sda.ServiceProcessor
-	identities      map[string]*storage
+	Identities      map[string]*Storage
 	identitiesMutex sync.Mutex
```

Comment:
the field is public so I can do `Marshal` `Unmarshal` on it. The mutex is to access it from the service.